lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -5481,6 +5499,10 @@ ParseResult FIRCircuitParser::parseToplevelDefinition(CircuitOp circuit,
     if (getToken().getKind() == FIRToken::kw_module)
       return parseModule(circuit, /*isPublic=*/true, indent);
     return emitError(getToken().getLoc(), "only modules may be public");
+  case FIRToken::kw_simulation:
+    if (requireFeature({4, 2, 0}, "simulation tests"))
```

Comment:
I just released 4.2.0 with some minor fixes around `%c`. We're into 5.0.0 territory given that I have merged the `{{}}` work. If you want to get the simulation tests onto a 4.3.0, I can do that for you. Otherwise, I think this is going to be 5.0.0.

Reply:
Does the `{{}}` break backwards compatibility? If that's the case, 5.0.0 sounds great! Otherwise, if existing 4.2.0 FIR would continue to work with `{{}}`, 4.3.0 might make more sense. I'll bump the `nextFIRVersion` to whatever we decide.

Reply:
Should I switch the `{{}}` feature to `nextFIRVersion` as well for consistency? To be bumped/replaced once we cut the spec release?